src/main/java/com/qaprosoft/carina/demo/phpTravels/components/Table.java

```diff
@@ -41,6 +41,12 @@ public void deleteBooking(String id) {
             }
         }
     }
-
-
+    public boolean getBookingIdExist(String id) {
+        for(TableRow row : rows) {
+            if(row.getBookingId().equals(id)) {
+                return false;
```

Comment:
your method will not work correctly because it'll return false if first row id doen't equal